src/datagrid/DataGrid.tsx

```diff
@@ -380,6 +580,7 @@ export class DataGrid extends React.PureComponent<DataGridProps, DataGridState>
 
```

Comment:
Please remove children from the props in render if it is not required

Reply:
done